de_docs/getting-started/beginner-guide.md

```diff
@@ -41,7 +41,7 @@ To get started, choose an option for installations available for your operating
 
 * [Paymetheus](/getting-started/install-guide.md#paymetheus)
 * [Command Line Suite](/getting-started/install-guide.md#dcrinstall)
-* [Decrediton (BETA)](https://github.com/decred/decrediton) - NOTE: Install instructions currently limited to the Github README.
+* [Decrediton (BETA)](https://github.com/decred/decrediton) - NOTE: Install instructions currently limited to the GitHub README.
```

Comment:
I'm seeing now that this is from the de_docs. Don't think I made those changes to the translated docs when I revised the original. My fault. This is probably fine for now.